internal/updater/install_test.go

```
@@ -0,0 +1,354 @@
+	dir := t.TempDir()
+	livePath := filepath.Join(dir, "bridge")
+	os.WriteFile(livePath, []byte("OLD"), 0o755)
+
+	rel := Release{
+		TagName: "v0.2.0",
+		Assets: []ReleaseAsset{
+			{Name: fix.archiveName, BrowserDownloadURL: tampered.URL + "/asset/" + fix.archiveName},
+			{Name: "checksums.txt", BrowserDownloadURL: tampered.URL + "/asset/checksums.txt"},
+		},
+	}
+	fix.releaseJSON, _ = json.Marshal(rel)
```

golangci-lint (2.11.4)

[error] Error return value of `encoding/json.Marshal` is not checked

(errchkjson)